libs/model/src/aggregates/token/CreateToken.command.ts

```diff
@@ -1,84 +1,45 @@
-import { InvalidState, logger, type Command } from '@hicommonwealth/core';
-import {
-  commonProtocol,
-  getErc20TokenInfo,
-  getLaunchpadTokenCreatedTransaction,
-} from '@hicommonwealth/evm-protocols';
+import { type Command } from '@hicommonwealth/core';
+import { commonProtocol } from '@hicommonwealth/evm-protocols';
 import * as schemas from '@hicommonwealth/schemas';
 import { BalanceSourceType } from '@hicommonwealth/shared';
 import { z } from 'zod';
 import { models } from '../../database';
 import { authRoles } from '../../middleware';
-import { mustExist } from '../../middleware/guards';
-
-const log = logger(import.meta);
 
 export function CreateToken(): Command<typeof schemas.CreateToken> {
   return {
     ...schemas.CreateToken,
     auth: [authRoles('admin')],
-    body: async ({ actor, payload }) => {
+    body: async ({ payload }) => {
       const {
-        chain_node_id,
+        creator_address,
+        token_address,
+        namespace,
+        launchpad_liquidity,
+        name,
+        symbol,
+        total_supply,
+        eth_chain_id,
         community_id,
-        transaction_hash,
         description,
         icon_url,
       } = payload;
 
-      const chainNode = await models.ChainNode.scope('withPrivateData').findOne(
```

Comment:
We can't remove this. This is a command that is available from the client and as such we must validate the transaction hash. Remember, there are 2 paths:
1. CreateToken command which is called from the client to register a new token
2. The Launchpad token created event which is the "insurance policy" for eventual consistency of created tokens.

We can trust that the event has the correct data but we cannot trust that the client sends the correct data.